content/guides/mobile/ios/quick-start/4-make-api-call.md

```diff
@@ -0,0 +1,127 @@
+---
+type: quick-start
+hide_in_page_nav: true
+---
+
+# Make an API call
+
+With our Box and iOS applications created and configured with the **Box iOS
+SDK** we can now make our first call to Box APIs.
+
+Using our blank iOS application, we will create a button to trigger
+a request to Box to fetch the name of the currently authenticated user.
+
+<Message warning>
+  As you integrate Box API calls into your iOS app, this blocking method
+  should be replaced with proper task delegation and non-blocking actions.
+</Message>
+
+## Create a button 
+
+Within your Swift application in Xcode, load `ContentView.swift`. At the top of
+the file you will see a `struct` for `ContentView`, within which is a basic
+string that will be output to your iOS application if you run the app in an
+emulator.
+
+<ImageFrame border center>
+  ![ContentView struct](./first-call-contentview.png)
+</ImageFrame>
+
+We'll first replace the `Text` output line with a button to be able to trigger
+off the call to get the current user. Replace that line with the below button.
+
+```js
+Button(action: {
+  // Perform some action
+}) {
+  Text("Click to get current user")
+}
+.padding(10)
+.cornerRadius(20)
+.foregroundColor(.white)
+.background(Color.blue)
+```
+
+Our next step is to add an an action for the button, which will be our API call
+to Box.
+
+## Add an API call button action
+
+We need to do two things add add the call to the Box API, add the import for
```

Comment:
"When a user clicks the button, we want to fetch the user's details. To achieve this we need to do two things..."